dcpy/connectors/edm/metadata.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+import pytz
+
+from dcpy.utils import git
+
+
+def generate() -> dict[str, str]:
+    """Generates "standard" s3 metadata for our files"""
+    metadata = {
+        "date_created": datetime.now(pytz.timezone("America/New_York")).strftime(
```

Comment:
Feels nice to have these be NY time. But should we add timezone info so it parses easily?